content/docs/command-reference/check-ignore.md

```diff
@@ -0,0 +1,84 @@
+# check-ignore
+
+Check whether any given files or directories are excluded from DVC
+due to the patterns found in [`.dvcignore`](/doc/user-guide/dvcignore).
+
+## Synopsis
+
+```usage
+usage: usage: dvc check-ignore [-h] [-q | -v] [-d] [-n]
+                               targets [targets ...]
+
+positional arguments:
+  targets        Input files/directories to check ignore patterns.
+```
+
+## Description
+
+For each pathname given via the command-line , check whether the file is excluded 
+by `.dvcignore` output the path if it is excluded.
+
+## Options
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+- `-d`, `--details` - show the exclude pattern together with each target path.
+
+- `-n`, `--non-matching` - show the target paths which don’t match any pattern.
+  Only usable when `--details` is also employed
+
+## Examples
+
+First, let's create a `.dvcignore` file with some patterns in it, and some files
+to check against it.
+
+```dvc
+$ echo "file*\n\!file2" >> .dvcignore
+$ cat .dvcignore
+file*
+!file2
+$ touch file1 file2 other
+$ ls
+file1  file2 other
+```
+
+Then, let's check if these files would be excluded given our `.dvcignore` file:
+
+```dvc
+$ dvc check-ignore file1
+file1
+$ dvc check-ignore file2
+file2
+$ dvc check-ignore other
+$ dvc check-ignore file*
+file1
+file2 
+```
+
+If the `--details` option is used, a series of lines are printed using this format:
+`<source> <COLON> <linenum> <COLON> <pattern> <HT> <pathname>`
```

Comment:
```suggestion
`.dvcignore:<line number>:<pattern> <HT> <target path>`
```
- Why `<source>`? It's always `.dvcignore` right? Or because of nested ones? If so maybe use `<path/to/>.dvcignore`
- What's <HT>? Please replace with actual character.

Reply:
@karajan1001 actually I'm not sure what the answers are here, can you check it? Thanks

Reply:
@jorgeorpinel, 
Not always `.dvcignore`, might be `data/.dvcignore` as well. We didn't restrict `.dvcignore` to the root of the repo.

Reply:
OK so probably:
```suggestion
`<path/to/.dvcignore>:<line_num>:<pattern> | <target_path>`
```
> HT = horizontal tab, right?